### DISCUSSION TITLE
What is the recommended approach to call APIs concurrently in wave?

### DISCUSSION THREAD

**Discussion by dulajra** [h2oai/wave#1286] [Q&A] [answered]
I want to execute a bunch of API calls that can be executed in parallel to save time. What is the recommended method to do that in Wave?

**Comment by mturoci** [comment, marked as answer]
This is not Wave-related, but rather python-related.

**Reply by dulajra** [reply]
Great! Because the Wave framework depends on `asyncio` event loop can there be any consequences if we use multithreading to do that?
